target/scripts/postfix-wrapper.sh

```diff
@@ -20,11 +20,11 @@ trap "service postfix stop" SIGINT
 trap "service postfix stop" SIGTERM
 trap "service postfix reload" SIGHUP
 
-service postfix start
-sleep 5
+service postfix start &
+wait
```

Comment:
I think the `sleep` here was, because `service postfix start` exit (0) successfully pretty fast, however postfix needs some time to come up completely.

```
service postfix start &
wait
```

is equal to just `service postfix start` ?

I am not sure if it's related, but I read on a issue/PR this week, about our CI behaving "flacky" again. Maybe this is the reason?

Reply:
Good points -- I'll play around with it and see what I find.